dask/dataframe/core.py

```diff
@@ -1654,6 +1661,44 @@ def count(self, axis=None, split_every=False):
                 result.divisions = (self.columns.min(), self.columns.max())
             return result
 
+    @derived_from(pd.DataFrame)
+    def mode(self, axis=0, dropna=True, split_every=False):  # this is the series class
+        """Use a counter on each partition, sum the resulting counters, iterate over key,
```

Comment:
I don't think we need this docstring. We'll copy it from DataFrame.mode.

Reply:
docstring removed